starters/features/tailwind/package.json

```diff
@@ -2,8 +2,18 @@
   "description": "Use Tailwind into your Qwik app",
   "devDependencies": {
     "postcss": "^8.4.16",
-    "tailwindcss": "^3.1.8"
+    "tailwindcss": "^3.1.8",
+    "autoprefixer": "10.4.11"
   },
+  "nextSteps": [
```

Comment:
let's add this to __qwik__, we have all this kind of functionality there

Reply:
ok! mmm what do you mean ? Do I need to move this?

Reply:
inside:
`__qwik__`